app/integration/runtime/pom.xml

```diff
@@ -84,6 +84,10 @@
       <groupId>org.apache.camel</groupId>
       <artifactId>camel-spring</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.camel</groupId>
+      <artifactId>camel-mustache</artifactId>
+    </dependency>
```

Comment:
it is fine for now but we should include this dependency only if the integration has a template step. We may need to do the same for atlasmap.

Reply:
Absolutely. So I left this in simply for 7.2. After that, we are intending to add both velocity and framemaker support in these template. At that point, the library dependency would be dynamic based upon the user's choice of syntax.